File: dotnet/app/Models/ex020.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace app.Models
{
    public class ex020
    {
        public void OrdemDeapresentacao()
        {
            Console.Clear();

            Console.Write("primeiro aluno: ");
            string? n1 = Console.ReadLine();
            Console.Write("segundo aluno: ");
            string? n2 = Console.ReadLine();
            Console.Write("terceiro aluno: ");
            string? n3 = Console.ReadLine();
            Console.Write("quarto aluno: ");
            string? n4 = Console.ReadLine();

            List<string?> lista = new List<string?> { n1, n2, n3, n4 };

            Random random = new Random();
            int n = lista.Count;
            while (n > 1)
            {
                int k = random.Next(n--);
                string? temp = lista[n];
                lista[n] = lista[k];
                lista[k] = temp;
            }
            Console.WriteLine($"a ordem de apresentação sera [ {string.Join(", ", lista)} ]");
        }
    }
}

File: javascript/ex020.js
console.clear()


const n1 = 'mailson';
const n2 = 'julia';
const n3 = 'fernanda';
const n4 = 'barbara';

const lista = [n1, n2, n3, n4];

for (let i = lista.length - 1; i > 0; i--) {
    const j = Math.floor(Math.random() * (i + 1));
    [lista[i], lista[j]] = [lista[j], lista[i]];
}

console.log(`a ordem de apresentação sera [ ${lista} ]`);

